edx-olive-oc/edx-platform
# Compiled the JS translations for Studio

Adds the Studio version of the compiled static JS translation files, missed by https://github.com/edx-olive-oc/edx-platform/pull/10

[Both `lms` and `cms` `djangjs.js` files](https://github.com/edx/edx-platform/blob/master/pavelib/i18n.py#L93-L95) must be compiled to produce reliable static JS assets.

**Sandbox server**

* LMS: https://campus-dev.opencraft.hosting/
* Studio: https://studio-campus-dev.opencraft.hosting/

**Testing instructions**

This issue is erratic and hard to reproduce reliably on the browser or in a devstack, but the cause of the issue can be checked on the edxapp server itself.

To verify:
1. Shell into the active campus-dev.opencraft.hosting appserver 
1. Change to the `/edx/var/edxapp/staticfiles` dir
1. Run the following commands to identify the different `djangojs*.js` files for Hebrew and Arabic:
   ```bash
   pomegranited@campus-dev i-00af0237:/edx/var/edxapp/staticfiles$ find . -path \*he* -name \*djangojs*.js -exec ls -l {} \;
   -rw-r--r-- 1 edxapp edxapp 330795 Sep 11 07:02 ./js/i18n/he/djangojs.c53b28fbad81.js
   -rw-r--r-- 1 edxapp edxapp 330795 Sep 11 07:00 ./js/i18n/he/djangojs.js
   -rw-r--r-- 1 edxapp edxapp 330795 Sep 11 07:07 ./ceb233d/js/i18n/he/djangojs.c53b28fbad81.js
   -rw-r--r-- 1 edxapp edxapp 330795 Sep 11 07:03 ./ceb233d/js/i18n/he/djangojs.js
   pomegranited@campus-dev i-00af0237:/edx/var/edxapp/staticfiles$ find . -path \*/ar/* -name \*djangojs*.js -exec ls -l {} \;
   -rw-r--r-- 1 edxapp edxapp 408005 Sep 11 07:02 ./js/i18n/ar/djangojs.c7f85aeda2c4.js
   -rw-r--r-- 1 edxapp edxapp 408005 Sep 11 07:00 ./js/i18n/ar/djangojs.js
   -rw-r--r-- 1 edxapp edxapp 408005 Sep 11 07:07 ./ceb233d/js/i18n/ar/djangojs.c7f85aeda2c4.js
   -rw-r--r-- 1 edxapp edxapp 408005 Sep 11 07:03 ./ceb233d/js/i18n/ar/djangojs.js
```
1. Note that the chunk hash extension used by each language is the same for the LMS `./js/i18n/` and Studio `./ceb233d/js/i18n/` compiled assets.
1. Diff the `djangojs*.js` files for each language, and ensure that no changes exist between the files for a given language.

**Reviewers**
